packages/file-adapters/lib/local-file.js

```diff
@@ -3,9 +3,14 @@ const path = require('path');
 const mkdirp = require('mkdirp');
 
 module.exports = class LocalFileAdapter {
-  constructor({ src, path: inputPath }) {
+  constructor({ src, path: inputPath, getFilename: nameGetter }) {
```

Comment:
Probably no need to rename `getFilename` to `nameGetter` here.

Reply:
Done.